lib/prmd/templates/schemata.md.erb

```diff
@@ -21,6 +21,8 @@ Stability: `<%= schemata['stability'] %>`
 <%- if schemata['properties'] && !schemata['properties'].empty? %>
 ### Attributes
 
+<details>
```

Comment:
I think this should be a bit further up in the file? Otherwise if there are no properties, the opening details tag would be omitted if I'm not mistaken. So I think if we just moved this up to around line 20 it would be safer. Does that makes sense?

Reply:
Exactly!
Thank you for youre comment!